core/math/fixed/fixed.odin

```diff
@@ -88,17 +88,20 @@ div_sat :: proc(x, y: $T/Fixed($Backing, $Fraction_Width)) -> (z: T) {
 
 @(require_results)
 floor :: proc(x: $T/Fixed($Backing, $Fraction_Width)) -> Backing {
-	return x.i >> Fraction_Width
+	if x.i >= 0 {
+		return x.i >> Fraction_Width
+	}
```

Comment:
The brace alignmtnt in use by the core library is to put else on the same line as the closing brace of the `if`, by the way